EternalModLoader/BlangParser.cs

```diff
@@ -0,0 +1,171 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace EternalModLoader
+{
+    /// <summary>
+    /// BlangFile class
+    /// </summary>
+    public class BlangFile
```

Comment:
This shouldn't be here. You should reference the BlangParser library that you can find here: https://github.com/dcealopez/BlangParser

This way, if I implement changes to the BlangParser library, they will be automatically traslated to the mod loader.